cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/distributed/DistributedMapReduceTaskContextProvider.java

```diff
@@ -126,31 +105,6 @@ protected void shutDown() throws Exception {
   }
 
   private static Injector createInjector(CConfiguration cConf, Configuration hConf) {
-    return Guice.createInjector(
```

Comment:
I checked these. Some have just been bundled into modules.
For instance, there was previously the explicit binding of a Logging class. Now, its using the LoggingModule#getDistributedModules, which has the same binding in it.
There are a few unnecessary bindings, but based upon testing, they don't seem to hurt (the code is much cleaner this way).